src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/how-to-upgrade-from-lts-2023-to-lts-2025/how-to-upgrade-to-jakarta-ee-10.md

```diff
@@ -0,0 +1,193 @@
+See below versions description:
+- Java EE 7 - previously used by Nuxeo
+- Jakarta EE 8 - Ownership transfered to Eclipse Foundation, Maven dependency change
+- Jakarta EE 9 - Java package change
+- Jakarta EE 10 - First release with enhancements and changes under the Eclipse Fondation umbrella
+
+## Maven Dependency
+
+They are two kinds of dependencies in the Jakarta EE context, the Specifications and the Implementations.
+A cleanup was done to our dependency graph to keep only the ones needed by Nuxeo Server.
+
+### Specfications
+
+You need to update the specifications you may use in your project to the Jakarta ones, generally the artifactId ends 
+with `-api`, at least it is the case in Jakarta EE.
+
+Find the API dependencies with `javax` groupId and replace them by the Jakarta equivalent, see the ones used by Nuxeo Server:
+
+```xml
+<dependencies>
+  ..
+  <dependency>
+    <groupId>jakarta.activation</groupId>
+    <artifactId>jakarta.activation-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.annotation</groupId>
+    <artifactId>jakarta.annotation-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.el</groupId>
+    <artifactId>jakarta.el-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.inject</groupId>
+    <artifactId>jakarta.inject-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.mail</groupId>
+    <artifactId>jakarta.mail-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.persistence</groupId>
+    <artifactId>jakarta.persistence-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.servlet</groupId>
+    <artifactId>jakarta.servlet-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.servlet.jsp</groupId>
+    <artifactId>jakarta.servlet.jsp-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.servlet.jsp.jstl</groupId>
+    <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.transaction</groupId>
+    <artifactId>jakarta.transaction-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.validation</groupId>
+    <artifactId>jakarta.validation-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.ws.rs</groupId>
+    <artifactId>jakarta.ws.rs-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.xml.bind</groupId>
+    <artifactId>jakarta.xml.bind-api</artifactId>
+  </dependency>
+  <dependency>
+    <groupId>jakarta.xml.ws</groupId>
+    <artifactId>jakarta.xml.ws-api</artifactId>
+  </dependency>
+  ..
+</dependencies>
+```
+
+### Implementations
+
+The implementations have changed too, some have just been renamed to jakarta, others have been changed to use other vendor.
+
+Generallyn a project doesn't state the Implementations it is using, it might just depend on the Specifications. If your 
```

Comment:
```suggestion
Usually, a project doesn't state the Implementations it is using, it may depend on the Specifications. If your 
project is using implementations, you will need to update them, see below the one used by Nuxeo Server:
```